test/e2e/suite/command/manifest.go

```diff
@@ -367,3 +367,47 @@ var _ = Describe("Common registry users:", func() {
 		})
 	})
 })
+
+var _ = Describe("OCI image layout users:", func() {
+	prepare := func(from string, tag string) string {
+		tmpRoot := GinkgoT().TempDir()
+		cpPath := tmpRoot
+		if tag != "" {
+			cpPath = fmt.Sprintf("%s:%s", tmpRoot, tag)
+		}
+		ORAS("cp", from, Flags.ToLayout, cpPath).WithDescription("prepare image from registry to OCI layout").Exec()
+		return tmpRoot
+	}
```

Comment:
refactor to `prepare := func(tag string) string`

Reply:
done and added cases for bad paths.